jobs/JGLOBAL_PREP_OCEAN_OBS

```diff
@@ -1,21 +1,29 @@
 #!/bin/bash
 source "${HOMEgfs}/ush/preamble.sh"
-source "${HOMEgfs}/ush/jjob_header.sh" -e "prepoceanobs" -c "base prepoceanobs"
+source "${HOMEgfs}/ush/jjob_header.sh" -e "prepoceanobs" -c "base ocnanal prepoceanobs"
 
 
 ##############################################
 # Set variables used in the script
 ##############################################
 
+export COMIN_OBS="${DATA}"
+
 ##############################################
 # Begin JOB SPECIFIC work
 ##############################################
 
+# Add prep_marine_obs.py to PYTHONPATH
+export PYTHONPATH=${HOMEgfs}/sorc/gdas.cd/ush/soca:${PYTHONPATH}
 
 ###############################################################
 # Run relevant script
 
-# the relevant script goes here
+EXSCRIPT=${GDASPREPPY:-${HOMEgfs}/sorc/gdas.cd/scripts/exglobal_prep_ocean_obs.py}
```

Comment:
This script needs to be linked into the workflow by `sorc/link_workflow.sh` so that no calling script needs to look into gdas.cd for a script.

Reply:
I think that would break the CI testing for the task in GDASApp. @CoryMartin-NOAA ?

Reply:
What if we move the exglobal script from GDASApp into global-workflow? It imports a python module from GDASApp

Reply:
The exscripts should go to g-w, that is the convention, but we have delayed that for ocean DA scripts due to rapid prototype development. But I think we are approaching the point where they need moved to g-w.

Reply:
While true, I wasn't recommend they be moved just yet. Just that they should be sym-linked into place from gdas.cd. We will handle actually moving them into the repo later.